pkg/nmi/server/linuxServer.go

```diff
@@ -0,0 +1,83 @@
+// +build linuxos
+
```

Comment:
The server can just be one too, right?

Reply:
For server, I think we need to files, one is for linux syscall, and the other is for windows syscall. If we combine to one file, build will not work.